_meta/ecs-migration.yml

```diff
@@ -0,0 +1,239 @@
+# The ECS migration file contains the information about all the fields which are migrated to ECS in 7.0.
+# The goal of the file is to potentially have scripts on top of this information to convert visualisations and templates
+# based on this information in an automated way and to keep track of all changes which were applied.
+#
+# The format of the file is as following:
+#
+# - from: source-field-in-6.x
+#   to: target-filed-in-ECS
+#   index: true|false
+
+- from: beat.hostname
+  to: observer.hostname
+  index: true
+
+- from: beat.name
+  to: observer.type
+  index: true
+
+- from: beat.version
+  to: observer.version
+  index: true
+
+- from: context.service.agent.name
+  to: agent.name
+  index: true
+
+- from: context.service.agent.version
+  to: agent.version
+  index: true
+
+- from: context.system.architecture
+  to: host.architecture
+  index: true
+
+- from: context.system.hostname
+  to: host.hostname
+  index: true
+
+- from: context.system.ip
+  to: host.ip
+  index: true
+
+- from: context.system.platform
+  to: host.os.platform
+  index: true
+
+- from: context.request.method
+  to: http.request.method
+  index: true
+
+- from: context.request.http_version
+  to: http.version
+  index: true
+
+- from: context.request.body
+  to: http.request.body.content
```

Comment:
@graphaelli so what do we do here?
keep `body.content` as in the last ECS beta and update 6.6, or go with `body` ?

Reply:
Since this field isn't indexed there's nothing to do in 6.6, right?  We should be able to move wherever we like in 7.0 and either a reindex of a 6.x index will move it, or the UI will smooth over the difference between the 6.x and 7.0 locations when it is displayed.

Reply:
I think only the response body is indexed, can you confirm?